.github/workflows/python.yml

```diff
@@ -2,8 +2,9 @@ name: Python
 
 on:
   push:
-    branches: [ master, python, experiments, stateful-c-lib ]
+    branches: [ master, python, experiments, iscgar/python-ci-fix ]
     tags: [ v* ]
+  workflow_dispatch:
```

Comment:
What does this do?

Reply:
Allows the workflow to be manually triggered from the UI. See my comment in https://github.com/solvespace/solvespace/pull/1530#issuecomment-2676343228